build.gradle

```diff
@@ -174,6 +174,13 @@ repositories {
         name = 'ModMaven'
         url = 'https://modmaven.k-4u.nl'
     }
+    maven {
+        name = 'CurseMaven'
+        url = 'https://www.cursemaven.com'
+        content {
+            includeGroup 'curse.maven'
+        }
+    }
```

Comment:
Is jade available anywhere else? I really don't want to have to use cursemaven

Reply:
If it is, their documentation doesn't mention it. It specifically tells you to use CurseMaven here: https://jademc.readthedocs.io/en/latest/plugins19/getting-started/

Reply:
please use the modrinth maven instead